### PR TITLE
[1194] Added accredited provider id to provider

### PR DESCRIPTION
**Pull request by defong**
### Context
Accredited provider id
### Changes proposed in this pull request
Add accredited provider id

### Guidance to review
skeleton code

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally
- [x] Inform data insights team due to database changes

### CODE REVIEW

**Comment by defong** [comment]
[Update existing provider form](https://trello.com/c/pqPIn7dh/1194-update-existing-provider-form)

**Review by goodviber — COMMENTED** [review]
would this need to be referenced on the api docs etc?

**Comment by darokel** [comment]
> > would this need to be referenced on the api docs
> 
> Nope...
> 
> @darokel do you have perception for exposing this field on the api?

If we want register to exclusively use the api to retrieve these APs then we would need to expose it. Saying that I don't think we need to do it right now as we don't (yet) have that comprehensive list of APs with their ids.

File: db/migrate/20230413112141_add_accredited_provider_id_to_provider.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAccreditedProviderIdToProvider < ActiveRecord::Migration[7.0]
+  def change
+    add_column :provider, :accredited_provider_id, :integer
```
Comment (darokel) [inline db/migrate/20230413112141_add_accredited_provider_id_to_provider.rb:5]: Do we need an index for this?

Comment (darokel) [reply db/migrate/20230413112141_add_accredited_provider_id_to_provider.rb:5]: yup fair enough, just wondered whether we'd ever search on it but probably not yet

**Review by darokel — APPROVED** [review]

File: db/migrate/20230413112141_add_accredited_provider_id_to_provider.rb (hunk 1 above)
Comment (defong) [reply db/migrate/20230413112141_add_accredited_provider_id_to_provider.rb:5]: Cool lets stick with YAGNI until there is a plausible future for it :smile_cat:

**Pull request merged**